jenkins-jobs/scripts/report.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+OPTS=`getopt -o c:e:r:a: --long connector:,env-file:,results-folder:,attributes: -n 'parse-options' -- "$@"`
+if [ $? != 0 ] ; then echo "Failed parsing options." >&2 ; exit 1 ; fi
+eval set -- "$OPTS"
+
+while true; do
+  case "$1" in
+    -c | --connector)           CONNECTOR=$2;                   shift; shift ;;
+    -e | --env-file )           ENV_FILE=$2;                    shift; shift ;;
+    -r | --results-folder )     RESULTS=$2;                     shift; shift ;;
+    -a | --attributes )         ATTRIBUTES=$2;                  shift; shift ;;
+    -h | --help )               PRINT_HELP=true;                shift ;;
+    -- ) shift; break ;;
+    * ) break ;;
+  esac
+done
+
+echo "Reporting to report portal!"
+
+docker login quay.io -u "$QUAY_USERNAME" -p "$QUAY_PASSWORD"
```

Comment:
personally I would omit this and assume the system is logged in prior to calling this script

Reply:
Which would also mean adding the login to jobs